pdns/dnsdist.cc

```diff
@@ -2091,19 +2090,15 @@ static void healthChecksThread()
       dss->prev.reuseds.store(dss->reuseds.load());
       
       for(IDState& ids  : dss->idStates) { // timeouts
-        int origFD = ids.origFD;
-        if(origFD >=0 && ids.age++ > g_udpTimeout) {
-          /* We set origFD to -1 as soon as possible
+        int64_t usageIndicator = ids.usageIndicator;
```

Comment:
Maybe use `.load()` ?

Reply:
Isn't that equivalent? Or do you think the use of `.load()` more readable, perhaps?

Reply:
It is. That was just to match https://github.com/PowerDNS/pdns/pull/8154/files#diff-41f3f8b6ff377ad8fb04d1a7a307a50eR547